apps/ledger-live-mobile/src/screens/ReceiveFunds/03-Confirmation.tsx

```diff
@@ -204,6 +206,17 @@ function ReceiveConfirmationInner({
     }
   }
 
+  let CustomConfirmationAlert;
+  if (
+    currency.type === "CryptoCurrency" &&
+    Object.keys(byFamilyPostAlert).includes(currency.family)
+  ) {
+    CustomConfirmationAlert =
+      currency.type === "CryptoCurrency"
```

Comment:
This is already checked in the if just above.

Reply:
I have to have it, otherwise typescript wont be happy